aspnetcore/release-notes/aspnetcore-10/includes/blazor.md

```diff
@@ -417,8 +403,293 @@ The Blazor project template now includes a `NotFound.razor` page by default. Thi
+
+`OrderPage.razor`:
+
+```razor
+<EditForm Model="Model">
+    <DataAnnotationsValidator />
+
+    <h3>Customer Details</h3>
+    <div class="mb-3">
+        <label>
+            Full Name
+            <InputText @bind-Value="Model!.Customer.FullName" />
+        </label>
+        <ValidationMessage For="@(() => Model!.Customer.FullName)" />
+    </div>
+
+    @* ... form continues ... *@
+</EditForm>
+
+@code {
+    public Order? Model { get; set; }
+
+    protected override void OnInitialized() => Model ??= new();
+
+    // ... code continues ...
+}
+```
+
+The requirement to declare the model types outside of Razor components (`.razor` files) is due to the fact that both the new validation feature and the Razor compiler itself are using a source generator. Currently, output of one source generator can't be used as an input for another source generator.
+
+### Custom Blazor cache and `BlazorCacheBootResources` MSBuild property removed
+
+Now that all Blazor client-side files are fingerprinted and cached by the browser, Blazor's custom caching mechanism and the `BlazorCacheBootResources` MSBuild property have been removed from the framework. If the client-side project's project file contains the MSBuild property, remove the property, as it no longer has any effect:
+
+```diff
+- <BlazorCacheBootResources>...</BlazorCacheBootResources>
+```
+
+For more information, see <xref:blazor/host-and-deploy/webassembly/bundle-caching-and-integrity-check-failures?view=aspnetcore-10.0>.
+
+### Web Authentication API (passkey) support for ASP.NET Core Identity
+
+[Web Authentication (WebAuthn) API](https://developer.mozilla.org/docs/Web/API/Web_Authentication_API) support, known widely as *passkeys*, is a modern, phishing-resistant authentication method that improves security and user experience by leveraging public key cryptography and device-based authentication. ASP.NET Core Identity now supports passkey authentication based on WebAuthn and FIDO2 standards. This feature allows users to sign in without passwords, using secure, device-based authentication methods, such as biometrics or security keys.
+
+The Preview 6 Blazor Web App project template provides out-of-the-box passkey management and login functionality.
+
+Migration guidance for existing apps will be published for the upcoming release of Preview 7, which is scheduled for mid-August.
+
+### Circuit state persistence
+
+Server-side Blazor apps (Blazor Web Apps, Blazor Server) during server-side rendering can now persist a user's session (circuit) state when the connection to the server is lost for an extended period of time or proactively paused, as long as a full-page refresh isn't triggered. This allows users to resume their session without losing unsaved work in the following scenarios:
+
+* Server restarts (but not app upgrades/deployments)
```

Comment:
Remove this bit, since this is not something we support out of the box. (Can be done through the current extensibility).